code/game/objects/effects/phased_mob.dm

```diff
@@ -32,7 +32,7 @@
 /// Removes [jaunter] from our phased mob
 /obj/effect/dummy/phased_mob/proc/eject_jaunter()
 	if(!jaunter)
-		CRASH("Phased mob ([type]) attempted to eject null jaunter.")
```

Comment:
Give me a chance today to look at how many times this runtime has occurred, and where, location wise, it has happened. I want to make sure we're not going to be hiding bugs.

Reply:
It's happened 0 times in the entire year of 2022, you're safe